packages/tools/examples/PolySegWasmContourToStackLabelmap/index.ts

```diff
@@ -0,0 +1,302 @@
+import { RenderingEngine, Enums, eventTarget } from '@cornerstonejs/core';
```

Comment:
Couldn't get this example working, not sure what is going wrong - it just never converts.

Reply:
I think i have seen this I will investigate more today

Reply:
I switched to Leo's code for containsPoint and seem to work based on my tests